setup.ps1

```diff
@@ -7,6 +7,23 @@ Param(
     [string]
     $composeFile = "$PSScriptRoot/docker-compose.yml"
 )
+Function Test-CommandExists {
+    Param ($command)
+
+    $oldPreference = $ErrorActionPreference
+    $ErrorActionPreference = 'Stop'
+
+    try {
+        if (Get-Command $command) {
```

Comment:
Werkt dit na een verse installatie van azure cli? Mogelijk dat PATH nog niet geupdated is namelijk.

Reply:
Correct, but you need to restart after the script anyway right?

Reply:
In het ergste geval zou die dan nogmaals de azure cli installeren dus dit lijkt mij verder wel beter dan dat die de installatie altijd zou overslaan

Reply:
I could add a restart message after the `az` install